graphdatascience/query_runner/gds_arrow_client.py

```diff
@@ -674,7 +676,11 @@ def _do_get(
             message=r"Passing a BlockManager to DataFrame is deprecated",
         )
 
-        return self._sanitize_arrow_table(arrow_table).to_pandas()  # type: ignore
+        if ServerVersion.from_string(pandas.__version__) >= ServerVersion(2, 0, 0):
```

Comment:
using ServerVersion here is a little confusing. Maybe we should rename the class?

Reply:
right we should name it so just SemanticVersion :+1

Reply:
#832